src/main/resources/google_checks.xml

```diff
@@ -48,6 +48,11 @@
 
     <module name="TreeWalker">
         <module name="OuterTypeFilename"/>
+        <module name="WhitespaceAfter">
```

Comment:
please move it to be right after/before `WhitespaceAround` to at least group Checks configs a bit.

Reply:
Done